turbo.json

```diff
@@ -1,19 +1,36 @@
 {
   "pipeline": {
+    "db-generate": {
+      "cache": true
+    },
+    "db-push": {
+      "cache": false
+    },
+    "dev": {
+      "dependsOn": ["^db-generate"],
+      "cache": false
+    },
     "build": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["^build", "^db-generate"],
```

Comment:
i removed the generate from postinstall and instead made it a dependency for build and dev, this makes it easier to cache too

Reply:
Cool! Good to know. 👍